package.tf

```diff
@@ -49,7 +49,7 @@ data "external" "archive_prepare" {
 resource "local_file" "archive_plan" {
   count = var.create && var.create_package ? 1 : 0
 
-  content              = data.external.archive_prepare[0].result.build_plan
+  content              = var.build_in_docker ? sensitive(data.external.archive_prepare[0].result.build_plan) : data.external.archive_prepare[0].result.build_plan
```

Comment:
The best solution would be to hide values ONLY if there are secrets inside. Function [`issensitive`](https://developer.hashicorp.com/terraform/language/functions/issensitive) became available in Terraform 1.8 but this module expects Terraform 1.0.